recipe/meta.yaml

```
@@ -1,20 +1,18 @@
 {% set name = "sklearn-contrib-lightning" %}
-{% set version = "0.5.0" %}
-{% set hash_val = "dca6cf1bea09cf77cf052119f4a43093" %}
+{% set version = "0.6.0" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://github.com/scikit-learn-contrib/lightning/archive/{{ version }}.tar.gz
-  md5: {{ hash_val }}
+  git_url: https://github.com/scikit-learn-contrib/lightning.git
+  git_rev: f05855f4ef9e7415c9afd16a033f2d2a4c192848
```

How come you changed to a git hash? It seems the source ***is*** available from pypi. I think you're more likely to get a quick merge if you use the pypi sources.
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
  sha256: ec21510ebc4f472a757b441eccb796398b193966c431b8178a1d52509d73a76e
```

---

@dhirschfeld Thanks for looking. Here's why:

- This recipe uses makefile-based build, and Makefile is not included in pip tarball.
- This recipe previously used github tarball linked to a tag, and a tag doesn't exist for 0.6.0.

I've tried fixing the recipe so we can get a package out asap (one might try and go fix the above two problems, but that might take an indefinite amount of time).

---

(Updated the comment above; if you see commit history, I actually did try pip tarball in one of the attempts)